packages/web/src/components/admin_page/admin.css

```diff
@@ -113,9 +113,10 @@
     flex-direction: row;
     flex-wrap: wrap;
     justify-content: center;
-    align-items: center;
-    width: 80vw;
-    margin-left: 10vw;
+    align-items: flex-start;
```

Comment:
what's the idea here?

Reply:
The jobs card in the admin panel had align-items: center which made them look unaligned correctly due to each card having different text lengths (see image below). So the align-items: flex-start at least aligns the cards in the same row at the top. I did this for aesthetic purposes for people from other departments to understand the cards easier. You can also see the difference if you run this PR locally and check [thehub-aubg](https://thehub-aubg.com/admin/dashboard/jobs) for comparison. 

![image](https://user-images.githubusercontent.com/77395265/227772984-c7b97e10-2eab-4317-a9b0-cfa94c13c4e3.png)

Reply:
cool